City-of-Helsinki/berth-reservations-admin
# VEN-944 | Batch printing customer addresses on A4 papers

## Description :sparkles:

Adds ability to generate sticker PDFs in the unmarked ws notice list page. Additionally refactors batch offer sending under the same list tool component.

### Closes :no_good_woman:

[VEN-944](https://helsinkisolutionoffice.atlassian.net/browse/VEN-944)

## Testing :alembic:

### Automated tests :gear:️

- Added `src/common/listActions/__test__/ListActions.test.tsx`
- Updated relevant tests and snapshots

### Manual testing :construction_worker_man:

- Open ws notices list page
- Select rows
- Choose "Tulosta tarrakirjeet" from "Toiminnot" dropdown
- If some of the selected notices do not have a sticker number, an error toast should appear when clicking the action button, otherwise it should be possible to save the generated pdf to disk or open it in the browser
